docs/docs/guides/labs/dg/incrementally-adopting-dg/migrating-project.md

```diff
@@ -13,63 +13,85 @@ This guide is only relevant if you are starting from an _existing_ Dagster proje
 
 :::
 
-Let's begin with a Dagster project that has some assets:
+We have a basic existing Dagster project and have been using `pip` to manage
+our Python environment. Our project defines a Python package with a `setup.py`
+and a single Dagster asset. The asset is exposed in a top-level `Definitions`
+object in `my_existing_project/definitions.py`.
 
 <CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/1-tree.txt" />
 
 ## Install dependencies
 
 ### Install the `dg` command line tool
 
-We'll need to install the `dg` command line tool, which is used to scaffold components. We recommend installing `dg` globally using the [`uv`](https://docs.astral.sh/uv/getting-started/installation/) package manager; it can also be installed using `pip`.
+Let's start with a fresh Python virtual environment (you may already have one):
 
-<CliInvocationExample contents="uv tool install dagster-dg" />
+<CliInvocationExample contents="python -m venv .venv && source .venv/bin/activate" />
 
-### Install `dagster-components`
+We'll need to install the `dg` command line tool. You can install it into the
+virtual environment using `pip`:
 
-Next, we'll need to install the `dagster-components` package.
+<CliInvocationExample contents="pip install dagster-dg" />
 
-Though this is optional, we generally recommend using a separate virtual environment for each project, which can be accomplished using `uv`:
+:::note
+For simplicity in this tutorial, we are installing the `dg` executable into the
+same virtual environment as our project. However, we generally recommend
+installing `dg` globally using the
+[`uv`](https://docs.astral.sh/uv/getting-started/installation/) package manager
+via `uv tool install dagster-dg`. This will install `dagster-dg` into an
+isolated environment and make it a globally available executable. This makes it
+easier to work with multiple projects using `dg`.
+:::
 
-<Tabs>
-    <TabItem value='before' label='Using uv virtual environment'>
-        <CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/3-uv-venv.txt" />
+### Install `dagster-components`
 
-        Then, we can use `uv sync` to install the dependencies from our `pyproject.toml`, and then install the `dagster-components` package:
-        <CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/4-uv-freeze.txt" />
-    </TabItem>
-    <TabItem value='after' label='Using pip'>
-        <CliInvocationExample contents="pip install dagster-components" />
-    </TabItem>
+Next, we'll need to add `dagster-components` as a dependency of our project. Add it to `install_requires` in `setup.py`:
 
-</Tabs>
+<CodeExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/2-setup.py" language="python" title="setup.py" />
+
+Now we install (or reinstall) our project into the active virtual environment:
```

Comment:
I would qualify this with: "If you are using editable installs"

Reply:
We would need to reinstall it regardless of whether we're using editable or not in order to get the new `dagster-components` dep installed